### PR TITLE
Add xmovie

### PR DESCRIPTION
**Pull request by jbusecke**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jbusecke** [comment]
Dear @conda-forge/help-python,  I am getting a message that a feedstock already exists for `xmovie`, but I cant find it [here](https://github.com/conda-forge?q=xmovie&type=&language=&sort=) or in the [staged_recipes PRs](https://github.com/conda-forge/staged-recipes/pulls?q=xmovie). Could you advise me how to resolve this? Many thanks.

**Comment by dopplershift** [comment]
If you look at the `recipe/xmovie/meta.yaml` that's actually uploaded to this PR, it looks completely like the template and only mentions "simplejson".

**Comment by jbusecke** [comment]
Sorry for being an absolute idiot and not checking. I modified the example recipe by accident. Sorry for wasting time here.

**Comment by dopplershift** [comment]
No worries, I ran through the same checks looking for an existing feedstock until I decided to look at `meta.yaml` for a clue, and got a big one. 😆

**Comment by jbusecke** [comment]
Alright. I think I have this all ironed out, and this should be ready to merge. Thanks for the assist @dopplershift!

**Review by dopplershift — CHANGES_REQUESTED** [review]
Looks pretty good. Just need this from the top of the template:

> Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe

**Comment by jbusecke** [comment]
Ah right. Thanks again.

File: recipes/xmovie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a7bbe3344da7d8349979ecc909ce74a0c5a7bb25e5cfcd07866a6ba562462497
+
+build:
+  number: 0
```
Comment (dopplershift) [inline recipes/xmovie/meta.yaml:13]: Need to remove this duplicated `build` section.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Missed this before. Everything else looks ready to go.

**Comment by jbusecke** [comment]
Fixed that duplicate, @dopplershift. Sorry for the added trouble on this. I owe you 🍺/☕️ in a not to distant future where I could visit Boulder

**Comment by dopplershift** [comment]
No worries! Though I'm definitely not turning down opportunities to have beer with people in that not-too-distant future.

File: recipes/xmovie/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (dopplershift) [inline recipes/xmovie/meta.yaml:13]: I have bad news...at some point you removed the `noarch: Python`

**Review by dopplershift — CHANGES_REQUESTED** [review]

**Review by dopplershift — APPROVED** [review]
🎉

**Pull request merged**